### PR TITLE
Changes to Templates

### PR DESCRIPTION
**Pull request by jakub-pomykala**
Added a "significant bug fixes" section to GA template.
Changed all occurances of "Java SE 14" to "Java SE 15"

### CODE REVIEW

File: templates/release-post.adoc [hunk 1]
```diff
@@ -217,6 +217,22 @@ For more details, check the LINK[LINK_DESCRIPTION].
 // // // // // // // //
 
 
+[#bugs]
+== Significant bugs fixed in this release
+
+
+We’ve spent some time fixing bugs. The following sections describe just some of the issues we resolved in this release. If you’re interested, here’s the full list of fixed bugs in 20.0.0.RELEASE_VERSION.
```
Comment (austin0) [inline templates/release-post.adoc]: `some of the issues we resolved in this release` -> `some of the issues resolved in this release`

or

`some of the issues we resolved in this release` -> `some of the issues we've resolved in this release`

**Review by lauracowen — DISMISSED** [review]
I agree with Austin's comment.
Otherwise, LGTM

File: templates/release-post.adoc (hunk 1 above)
Comment (jakub-pomykala) [reply templates/release-post.adoc]: Changed to the first one.

**Review by austin0 — DISMISSED** [review]
LGTM

**Review by austin0 — APPROVED** [review]
LGTM

**Pull request merged**